staging/prometheus-operator/patch/mesosphere/patch/11_use_existing_storage_definitions.patch

```diff
@@ -0,0 +1,94 @@
+commit 7c01b4a4e11cd75a4bb2f8c7d655ff3b789d0dfb
+Author: Joe Julian <me@joejulian.name>
+Date:   Mon Mar 15 17:44:27 2021 -0700
+
+    use exiting storage config if it already exists
+
+    If either prometheus or alertmanager resources exist and have a storage
+    attribute defined in their spec, reuse that instead of what's defined
+    in the values.
+
+    This prevents the accidental redefinition of the volumeClaimTemplate and
+    the abaondomnet of old data.
+
+diff --git a/staging/prometheus-operator/templates/NOTES.txt b/staging/prometheus-operator/templates/NOTES.txt
+index 4ef1bc3c8..f08a8afaa 100644
+--- a/staging/prometheus-operator/templates/NOTES.txt
```

Comment:
is this stuff supposed to be here?

Reply:
yeah, it's a patch file that gets applied on top of the upstream code so we can apply our changes on top of the fork.